File: Latihan B/Java/bola.java
public class bola {
    // deklarasi variabel
    private int tahun;
    private String nama;
    private String negara;
    private String pemain;

    public bola(String nama, String negara, int tahun, String pemain){
        this.nama = nama;
        this.negara = negara;
        this.tahun = tahun;
        this.pemain = pemain;
    }

    public bola(){
    }

    /*
        Setter dan Getter
    */ 

    public int getTahun() {
        return tahun;
    }
    public void setTahun(int tahun) {
        this.tahun = tahun;
    }

    public String getNama() {
        return nama;
    }
    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getNegara() {
        return negara;
    }
    public void setNegara(String negara) {
        this.negara = negara;
    }

    public String getPemain() {
        return pemain;
    }
    public void setPemain(String pemain) {
        this.pemain = pemain;
    } 

    // method untuk menampilkan output
    public void keluaran(){
        System.out.println("==============================================");
        System.out.println("Nama tim : " + this.nama);
        System.out.println("Negara asal : " + this.negara);
        System.out.println("Tahun berdiri : " + this.tahun);
        System.out.println("Pemain : " + this.pemain);
    }
}

File: latihan1/Latihan B/PHP/bola.php
<?php
    class bola{
        // atribut
        private $Nama = "Nama";
        private $Negara = "Negara";
        private $Tahun = 0;
        private $Pemain = "Pemain";

        // constructor
        public function __construct($Nama="Nama", $Negara="Negara", $Tahun=0, $Pemain="Pemain"){
            $this->Nama = $Nama;
            $this->Negara = $Negara;
            $this->Tahun = $Tahun;
            $this->Pemain = $Pemain;
        }

        /*
            Setter dan Getter
        */ 

        public function setNama($Nama){
            $this->Nama = $Nama;
        }
        public function getNama(){
            return $this->Nama;
        }
        
        public function setNegara($Negara){
            $this->Negara = $Negara;
        }
        public function getNegara(){
            return $this->Negara;
        }

        public function setTahun($Tahun){
            $this->Tahun = $Tahun;
        }
        public function getTahun(){
            return $this->Tahun;
        }

        public function setPemain($Pemain){
            $this->Pemain = $Pemain;
        }
        public function getPemain(){
            return $this->Pemain;
        }

        // untuk menampilkan keluaran
        public function print(){
            echo "==============================================<br>";
            echo "Nama Tim: ".$this->Nama."<br>";
            echo "Negara asal : ".$this->Negara."<br>";
            echo "Tahun berdiri: ".$this->Tahun."<br>";
            echo "Pemain : ".$this->Pemain."<br>";
        }          

        // destructor
        public function __destruct(){
        }
    }
?>